File: dongyimai_sellergoods_interface/src/main/java/com/offcn/sellergoods/service/GoodsDescService.java
package com.offcn.sellergoods.service;

import com.offcn.entity.PageResult;
import com.offcn.pojo.TbGoodsDesc;

import java.util.List;

public interface GoodsDescService {

	//findALL
	public List<TbGoodsDesc> findAll();
	
	//findPage
	public PageResult findPage(int pageNum, int pageSize);

	//add
	public void add(TbGoodsDesc goods_desc);
	
	//update
	public void update(TbGoodsDesc goods_desc);
	
	//findOne
	public TbGoodsDesc findOne(Long goodsId);
	
	//delete
	public void delete(Long[] goodsIds);

	//条件、模糊查询+分页
	public PageResult findPage(TbGoodsDesc goods_desc, int pageNum, int pageSize);
	
}

File: dongyimai_manager_web/src/main/webapp/js/service/goodsDescService.js
//服务层
app.service('goodsDescService',function($http){
	    	
	//findAll
	this.findAll=function(){
		return $http.get('../goodsDesc/findAll.do');		
	}

	//findPage
	this.findPage=function(page,rows){
		return $http.get('../goodsDesc/findPage.do?page='+page+'&rows='+rows);
	}

	//findOne
	this.findOne=function(id){
		return $http.get('../goodsDesc/findOne.do?id='+id);
	}

	//add
	this.add=function(entity){
		return  $http.post('../goodsDesc/add.do',entity );
	}

	//update
	this.update=function(entity){
		return  $http.post('../goodsDesc/update.do',entity );
	}

	//dele
	this.dele=function(ids){
		return $http.get('../goodsDesc/delete.do?ids='+ids);
	}

	//条件、模糊查询+分页
	this.search=function(page,rows,searchEntity){
		return $http.post('../goodsDesc/search.do?page='+page+"&rows="+rows, searchEntity);
	}

});